config/config.exs

```diff
@@ -30,3 +30,9 @@ use Mix.Config
 #     import_config "#{Mix.env}.exs"
 
 config :ex_wrapplog, level: :debug
```

Comment:
Yes its default to debug. It can be set to anything by setting the config. I am not sure what to set the default value but in all the elixir services I have set to info. So it is just a config